src/main.cpp

```diff
@@ -179,6 +201,14 @@ void slice(int argc, char **argv)
                     case 'v':
                         cura::increaseVerboseLevel();
                         break;
+#ifdef _OPENMP
+                    case 'm':
+                        str++;
+                        n_threads = *str - '0';
+                        n_threads = std::max(1, n_threads);
+                        omp_set_num_threads(n_threads);
+                        break;
```

Comment:
I have a feeling we have duplicate code here. Shouldn't we lift this out of the command-line-options bit and the socket connection bit and into a separate place to parse these arguments?